packages/twenty-server/src/engine/twenty-orm/base.workspace-entity.ts

```diff
@@ -36,4 +37,14 @@ export abstract class BaseWorkspaceEntity {
     defaultValue: 'now',
   })
   updatedAt: Date;
+
+  @WorkspaceField({
+    standardId: BASE_OBJECT_STANDARD_FIELD_IDS.deletedAt,
+    type: FieldMetadataType.DATE_TIME,
+    label: 'Deleted at',
+    description: 'Date when the record was deleted',
+    icon: 'IconCalendarMinus',
+  })
+  @WorkspaceIsNullable()
+  deletedAt?: Date | null;
```

Comment:
When I look at a production workspace it seems this column was created already by I don't understand why!? I'm missing something

Reply:
Hmm, on custom and standard objects ? I'm not having them on my computer

Reply:
@FelixMalfait @magrinj probably because it's defined there as a default column for table creation https://github.com/twentyhq/twenty/blob/main/packages/twenty-server/src/engine/workspace-manager/workspace-migration-runner/utils/custom-table-default-column.util.ts
Originally I even put in the migration-runner in advance so it would have affected standard objects as well (and probably why you see some in prod) but it has been removed after many refactoring of this part I guess.

Reply:
that was my initial assumption but then why aren't position or name also created on every table?

Reply:
Even for recent standard objects from v0.23 I see `deletedAt` column but I don't see `position` (e.g. NoteTarget)